.travis.yml

```diff
@@ -242,6 +249,30 @@ before_install:
   - if [ "$ASAN" == "On" ]; then export SANITIZER=Address; fi
   - if [ -n "$CLANG_VERSION" ]; then sudo PATH="${PATH}" CXX="$CXX" CC="$CC" ./install_libcxx.sh; fi
 
+  # Download and install Boost
+  # NOTE: Boost is only installed and tested for clang-trunk -std=c++1z without asan and without msan
```

Comment:
Yes you are right, that's just something that will get out of sync. I've added that to the commit message and removed it from here.